concourse-ci/tasks/version-calculator-task.yml

```diff
@@ -7,20 +7,25 @@ image_resource:
     repository: node
     tag: 18-slim
 
-caches:
-  - 
+# Cache directories are relative to the working directory of the task :/
```

Comment:
That doesn't really seem to make it faster .. will leave it here for now but I should improve this later on.